hack/boilerplate/boilerplate.py

```diff
@@ -176,7 +176,7 @@ def get_regexs():
     # Search for "YEAR" which exists in the boilerplate, but shouldn't in the real thing
     regexs["year"] = re.compile( 'YEAR' )
     # dates can be 2014, 2015, 2016, or 2017; company holder names can be anything
-    regexs["date"] = re.compile( '(2014|2015|2016|2017)' )
+    regexs["date"] = re.compile( '(2014|2015|2016|2017|2018)' )
```

Comment:
Is this in another PR? Also wth are we not doing this programatically.

Reply:
I got lazy :-(